vendor/k8s.io/kubernetes/test/e2e/node/pods.go

```diff
@@ -54,7 +54,7 @@ var _ = SIGDescribe("Pods Extended", func() {
 		/*
 			Release : v1.15
 			Testname: Pods, delete grace period
-			Description: Create a pod, make sure it is running. Using the http client send a 'delete' with gracePeriodSeconds=30. Pod SHOULD get deleted within 30 seconds.
+			Description: Create a pod, make sure it is running. Using the http client send a 'delete' with gracePeriodSeconds=30. Pod SHOULD get terminated within gracePeriodSeconds and removed from API server within a window.
 		*/
 		ginkgo.It("should be submitted and removed [Flaky]", func() {
```

Comment:
in the upstream PR the `[Flaky]` is dropped

Reply:
thanks for the review.  Restored now.  I've added a new commit to regenerate the bindata so tests don't fail